docs/reference/contributing/target/QuadSPI.md

```diff
@@ -29,6 +29,28 @@ To make sure your platform is ready for the upcoming changes, you need to implem
 
```

Comment:
Query: Can we delete or change the warning in line 7?

Reply:
The code was merged it looks like, should be good to remove that warning,

Reply:
@AnotherButler warning was removed